src/deroff.rs

```diff
@@ -761,7 +761,14 @@ impl Deroffer {
     //             return self.esc()
 
     fn number<'a>(&mut self, s: &'a str) -> Option<&'a str> {
-        unimplemented!()
+        //unimplemented!()
```

Comment:
I believe this line is not needed anymore.

Reply:
I've removed it, thanks.